qiskit/unroll/_jsonbackend.py

```diff
@@ -254,14 +254,16 @@ def drop_condition(self):
         self.creg = None
         self.cval = None
 
-    def start_gate(self, name, args, qubits, nested_scope=None):
+    def start_gate(self, name, args, qubits, nested_scope=None, extra_fields=None):
         """Begin a custom gate.
 
         name is name string.
         args is list of Node expression objects.
         qubits is list of (regname, idx) tuples.
         nested_scope is a list of dictionaries mapping expression variables
         to Node expression objects in order of increasing nesting depth.
+        extra_fields is a dictionary allowing the extension or overriding of
+        the gate instruction properties.
```

Comment:
Can you take the chance to change this into a proper docstring, even if technically not part of the PR?